scapy/layers/radius.py

```diff
@@ -1150,7 +1150,10 @@ def post_build(self, p, pay):
         return p
 
 
-bind_layers(UDP, Radius, sport=1812)
-bind_layers(UDP, Radius, dport=1812)
-bind_layers(UDP, Radius, sport=1813)
-bind_layers(UDP, Radius, dport=1813)
+bind_bottom_up(UDP, Radius, sport=1812)
+bind_bottom_up(UDP, Radius, dport=1812)
+bind_bottom_up(UDP, Radius, sport=1813)
+bind_bottom_up(UDP, Radius, dport=1813)
+bind_bottom_up(UDP, Radius, sport=3799)
+bind_bottom_up(UDP, Radius, dport=3799)
+bind_layers(UDP, Radius, sport=1812, dport=1812)
```

Comment:
```suggestion
bind_layers(UDP, Radius, sport=1813, dport=1813)
```

Reply:
The major port is 1812 used to authentication, 1813 is for accounting and 3799 for CoA. But, ok. I updated the branch.

Reply:
Feel free to do whatever is best. Just remember to update the test afterwards

Reply:
But, now is green. right?